src/Components/SaveButton.tsx

```diff
@@ -39,16 +40,23 @@ const RadioButtonItem = styled.button<{ saved: boolean }>`
 
 export const SaveButton = ({ id, saved }: SaveButtonProps): JSX.Element => {
   const { savedUpdate } = useContext(SaveContext)
+  const { notify } = useContext(NotificationsContext)
+
+  const handleClick = (e: React.MouseEvent) => {
+    e.stopPropagation()
+    const isSaved = savedUpdate(id)
+    notify({
+      msg: isSaved ? 'Service saved' : 'Service removed from saved',
```

Comment:
Maybe 'Added to saved list' : 'Removed from saved list' ? I don't think we refer to anything as a service outside of the code.

Reply:
Sure, updated